test/Kestrel.Core.Tests/ListenOptionsTests.cs

```diff
@@ -14,5 +15,13 @@ public void ProtocolsDefault()
             var listenOptions = new ListenOptions(new IPEndPoint(IPAddress.Loopback, 0));
             Assert.Equal(HttpProtocols.Http1, listenOptions.Protocols);
         }
+
+        [Fact]
+        public void Http2DisabledByDefault()
+        {
+            var listenOptions = new ListenOptions(new IPEndPoint(IPAddress.Loopback, 0));
+            Assert.Throws<NotSupportedException>(() => listenOptions.Protocols = HttpProtocols.Http1AndHttp2);
```

Comment:
Can you assert the exception message matches what's in CoreStrings.Http2NotSupported?